Assets/MRTK/Tests/PlayModeTests/FocusProviderRaycastTests.cs

```diff
@@ -56,8 +56,8 @@ public void SetupFocusProviderRaycastTests()
 
             pointer = new TestPointer();
             focusProvider.RegisterPointer(pointer);
-            
-            GameObject raycastTestPrefab = AssetDatabase.LoadAssetAtPath<GameObject>(Path.Combine("Assets", "MixedRealityToolkit.Tests", "PlayModeTests", "Prefabs", "FocusProviderRaycastTest.prefab"));
+
+            GameObject raycastTestPrefab = AssetDatabase.LoadAssetAtPath<GameObject>(AssetDatabase.GUIDToAssetPath("dc3b0cf66c0615e4c81d979f35d51eaa"));
```

Comment:
Did we do a deep scrub for AssetDatabase calls like this?

Reply:
yes. @keveleigh made an update that removed these in a previous PR. this is a new addition since that occurred.

Reply:
all play mode tests passed after this change.

Reply:
It might be good to leave a comment about which prefab this is meant to point to, in case the test needs updating in the future. (Reviewing on my phone, so let me know if i just missed it!)